README.md

```diff
@@ -48,26 +48,52 @@ CLX is available in a Docker container, by building from source, and through Con
 
 ### CLX Docker Container
 
-Pull the RAPIDS container and build for CLX.
+Prerequisites
+* NVIDIA Pascal™ GPU architecture or better
+* CUDA 9.2 or 10.0 compatible NVIDIA driver
+* Ubuntu 16.04/18.04 or CentOS 7
+* Docker CE v18+
+* nvidia-docker v2+
+
+Pull the RAPIDS container and build CLX image.
 
 ```aidl
 docker pull rapidsai/rapidsai-dev-nightly:0.11-cuda10.0-devel-ubuntu18.04-py3.7
-docker build -t clx .
+docker build -t clx:latest .
+```
+
+Start Container and Notebook Server
+
+#### Preferred - Docker CE v19+ and nvidia-container-toolkit
+```aidl
+docker run  --gpus '"device=0"' \
+  --rm -d \
+  -p 8888:8888 \
```

Comment:
@brhodes10 Do we use the port binding for ports 8787 and 8686 or can I remove them?

Reply:
Those port bindings I believe are for dask? Wanted to stay consistent with the RAPIDS container https://hub.docker.com/r/rapidsai/rapidsai in case the user wanted to use them as well.